modules/docs/markdown/04-codegen/01-customisation/08-wildcard.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_label: Wildcard imports
+title: Adjust Scala wildcard
+---
+
+Scala has a specific syntax for wildcard argument in types. In Scala 2, that was the underscore: `_`. But with Scala 3, this is changing. See this [page](https://docs.scala-lang.org/scala3/reference/changed-features/wildcards.html) for more information.
+
+Smithy4s now has a way for you to control that and the good thing is that you probably don't have to worry about it. If you're using Smithy4s via `mill` or `sbt`, then it's taken care of you. It can be overriden via the following keys:
```

Comment:
```suggestion
Smithy4s now has a way for you to control that, and the good thing is that you probably don't have to worry about it. If you're using Smithy4s via `mill` or `sbt`, then it's taken care of you. It can be overriden via the following keys:
```